Release Cognitive Services Data Plane SDKs Round I (#4955)
Azure/azure-sdk-for-js: sdk/cognitiveservices/cognitiveservices-translatortext/README.md

## An isomorphic javascript sdk for - TranslatorTextClient

This package contains an isomorphic SDK for TranslatorTextClient.

### Currently supported environments

- Node.js version 6.x.x or higher
- Browser JavaScript

### How to Install

```bash
npm install @azure/cognitiveservices-translatortext
```

### How to use

#### nodejs - Authentication, client creation and languages translator as an example written in TypeScript.

##### Install @azure/ms-rest-azure-js

```bash
npm install @azure/ms-rest-azure-js
```

##### Sample code

> The following sample lets you detect the text of input. The input is given in Chinese and the sample lets you detect it correctly. To know more, Refer the [Azure Documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/translator/)

Reviewer: ```suggestion
The following sample translates the given text which is in Chinese to English. To know more, refer to the [Azure Documentation on Translator](https://docs.microsoft.com/en-us/azure/cognitive-services/translator/)
```
